docs/database/database-spans.md

```diff
@@ -478,6 +480,43 @@ Semantic conventions for individual database systems or specialized instrumentat
 MAY specify a different `db.query.summary` format as long as produced summary remains
 relatively short and its cardinality remains low comparing to the `db.query.text`.
 
+## Propagating context to databases
+
+**Status**: [Development][DocumentStatus]
+
+Instrumentations SHOULD propagate the context information to the SQL queries following [sqlcommenter](https://google.github.io/sqlcommenter/spec/).
+
+### Recommended sqlcommenter attributes
+
+| Attribute              | Type   | Description                           | Require level                | Stability                                                      |
+|------------------------|--------|---------------------------------------|------------------------------|----------------------------------------------------------------|
+| `baggage.service.name` | string | Logical name of the service [1]       | `Conditionally Required` [2] | ![Development](https://img.shields.io/badge/-development-blue) |
+| `traceparent`          | string | The trace context of current span [3] | `Recommended` [4]            | ![Development](https://img.shields.io/badge/-development-blue) |
+
+**[1] `baggage.service.name`:** MUST be the same for all instances of horizontally scaled services. If the value was not specified, SDKs MUST fall back to `unknown_service:` concatenated with [process.executable.name](https://opentelemetry.io/docs/specs/semconv/attributes-registry/process/), e.g. `unknown_service:bash`. If `process.executable.name` is not available, the value MUST be set to `unknown_service`.
+
+[Baggage](https://www.w3.org/TR/baggage/) means you can pass data across services and processes, making it available to add to traces, metrics, or logs in those services.
+
+**[2] `baggage.service.name`:** SHOULD be set if `traceparent` cannot be injected due to performance issues.
+
+**[3] `traceparent`:** MUST be in the [text format](https://www.w3.org/TR/trace-context/#traceparent-header).
+
+**[4] `traceparent`:** `traceparent` has extremely high-cardinality. It's RECOMMENDED to propagate this info if the high-cardinality is safe for the database observabilty engine.
+
+**Examples:**
+
+- Query with `baggage.service.name`:
+
+  ```sql
+  SELECT * FROM songs /* baggage.service.name=music-player%3Aplay */
+  ```
+
+- Query with `baggage.service.name` and `traceparent`
+
+  ```sql
+  SELECT * FROM songs /* baggage.service.name=music-player%3Aplay, traceparent=00-0af7651916cd43dd8448eb211c80319c-b7ad6b7169203331-01 */
```

Comment:
Per the sqlcomment spec, the values should be wrapped with a `'` character, so I think this suggestion is a more accurate output (this applies to the previous example as well):

```suggestion
  SELECT * FROM songs /* baggage.service.name='music-player%3Aplay', traceparent='00-0af7651916cd43dd8448eb211c80319c-b7ad6b7169203331-01' */
```